tools/unitctl/README.md

```diff
@@ -51,11 +51,44 @@ desired.
 
 ## Features (Current)
 
+```
+CLI interface to the NGINX UNIT Control API
+
+Usage: unitctl [OPTIONS] <COMMAND>
+
+Commands:
+  instances  List all running UNIT processes
+  edit       Open current UNIT configuration in editor
+  import     Import configuration from a directory
+  execute    Sends raw JSON payload to UNIT
+  status     Get the current status of UNIT
+  listeners  List active listeners
+  help       Print this message or the help of the given subcommand(s)
+
+Options:
+  -s, --control-socket-address <CONTROL_SOCKET_ADDRESS>
+          Path (unix:/var/run/unit/control.sock), tcp address with port (127.0.0.1:80), or URL
+  -w, --wait-timeout-seconds <WAIT_TIME_SECONDS>
+          Number of seconds to wait for control socket to become available
+  -t, --wait-max-tries <WAIT_MAX_TRIES>
+          Number of times to try to access control socket when waiting [default: 3]
+  -h, --help
+          Print help
+  -V, --version
+          Print version
+```
+
 - Consumes alternative configuration formats Like YAML and converts them
+- Can convert output to multiple different formats.
 - Syntactic highlighting of JSON output
 - Interpretation of Unit errors with (arguably more) useful error messages
 
 ### Lists all running Unit processes and provides details about each process.
+Unitctl will detect and connect to running process of Unit on the host.
```

Comment:
```suggestion
Unitctl will detect and connect to running processes of Unit on the host.
```
Assuming it will connect to all of them.

"On the host" also implies it will not detect the docker ones, though ... are they still on the host?